poc-rails-template/serializer.mustache

```diff
@@ -0,0 +1,12 @@
+=begin
```

Comment:
I thought the templates went into `/modules/openapi-generator/..../resources/...` somewhere?

Reply:
I'm using the custom templates, and open-api-generator allows for specifying the template folder, so I wonder if we want to make the template in a top-level folder that's more explicit for us?